In-game emote binder

eps0003 opened the pull request:
## Status

**READY**

## Description

This adds an in-game menu which allows you to change the emote for each hotkey.

![image](https://user-images.githubusercontent.com/30195802/50675559-f83acd80-1042-11e9-844b-20ed356e4419.png)


Closes #154

## Steps to Test or Reproduce

1. Join a game
2. Open main menu by clicking escape key
3. Click "Bind Emotes" button
4. Change emotes
5. Respawn to update emotes

AsuMagic commented on Rules/CommonScripts/EmoteBinderMenu.as:
```diff
@@ -0,0 +1,166 @@
+void onMainMenuCreated(CRules@ this, CContextMenu@ menu)
+{
+	//add button to main menu
+	Menu::addContextItem(menu, getTranslatedString("Bind Emotes"), "EmoteBinderMenu.as", "void NewMenu()");
```
Maybe have a "bindings" parent menu for this so more things like that can be added in the same submenu? e.g. taunts

Blubahub commented:
Don't forget to make sure it'll safely read manually-edited EmoteBindings.cfg.

Also, you don't have all the in-game emotes, like clap, cool, wink, and so forth.

eps0003 commented:
- Added clap emote to emotes list
- Fixed issue with menu size when there's a perfect number of emotes
- Added @AsuMagic's suggestion to add a "Bindings" parent menu
